{
    "functionName": "test_we_throw_an_error_for_invalid_status_types",
    "className": null,
    "fileName": "/AutomatedTester_&_Bugsy/tests_&_test_bugs.py",
    "projectName": "repos",
    "Label": false,
    "isTest": true,
    "Body": "bug = Bug(**bug_return['bugs'][0])\ntry:\n    bug.status = 'foo'\n    assert False, 'Should have thrown an error about invalid type'\nexcept BugException as e:\n    assert str(e) == 'Message: Invalid status type was used Code: None'\n"
}